src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/FixCaseApiService.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.ecm.common.helpers.UtilHelper;
+import uk.gov.hmcts.ecm.common.model.ccd.CaseData;
+import uk.gov.hmcts.ecm.common.model.ccd.CaseDetails;
+import uk.gov.hmcts.ecm.common.model.multiples.SubmitMultipleEvent;
+import uk.gov.hmcts.ethos.replacement.docmosis.helpers.MultiplesHelper;
+import uk.gov.hmcts.ethos.replacement.docmosis.service.excel.MultipleCasesReadingService;
+
+import java.util.List;
+
+import static uk.gov.hmcts.ecm.common.model.helper.Constants.MULTIPLE_CASE_TYPE;
+
+@RequiredArgsConstructor
+@Service("fixCaseApiService")
+public class FixCaseApiService {
+
+    private final MultipleCasesReadingService multipleCasesReadingService;
+
+    @Value("${ccd_gateway_base_url}")
+    private String ccdGatewayBaseUrl;
+
+    public void checkUpdateMultipleReference(CaseDetails caseDetails, String userToken) {
+        CaseData caseData = caseDetails.getCaseData();
+        if (caseData.getEcmCaseType() != null && caseData.getEcmCaseType().equals(MULTIPLE_CASE_TYPE)) {
```

Comment:
You can improve this a little bit. The null check isn't required if you do this:
`if (MULTIPLE_CASE_TYPE.equals(caseData.getEcmCaseType(() {`

Reply:
Updated